### PR TITLE
SIMSBIOHUB-500: Removing more dead code

### PR DESCRIPTION
**Pull request by curtisupshall**
## Links to Jira Tickets

- https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-500

## Description of Changes

 - Removed all references to 'occurrence' and 'dwc'
 - Removed CSV transforming
 - Removed occurrence submitting + error handling

## Testing Notes
- All parts of the app should work like  normal

### CODE REVIEW

**Comment by NickPhura** [comment]
@curtisupshall Can you blow away the 2 dwc seeds as well (02 and 05 I believe).

File: api/src/constants/status.ts [hunk 1]
```diff
@@ -1,65 +1,3 @@
...
-  'FAILED_TO_GET_TEMPLATE_NAME_VERSION' = 'Missing Name or Version Number',
-  'FAILED_GET_VALIDATION_RULES' = 'Failed to Get Validation Rules',
-  'FAILED_PARSE_VALIDATION_SCHEMA' = 'Failed to Parse Validation Schema',
-  'UNSUPPORTED_FILE_TYPE' = 'Unsupported File Type',
-  'FOUND_VALIDATION' = 'Found Validation',
-  'SYSTEM_ERROR' = 'System Error'
-}
-
 // Message types that match the submission_message_type table
 export enum SUBMISSION_MESSAGE_TYPE {
```
Comment (NickPhura) [inline api/src/constants/status.ts:2]: Not critical, but there are probably some more statuses we could remove.

There are a handful that have no references anywhere else in the code. May as well purge them.

Comment (curtisupshall) [reply api/src/constants/status.ts:2]: Sure

Comment (curtisupshall) [reply api/src/constants/status.ts:2]: Done

**Review by NickPhura — APPROVED** [review]
Looks good
🐫

**Review by al-rosenthal — APPROVED** [review]
🍕

**Pull request merged**